### PR TITLE
Adding the managed tenants module mapping

### PR DESCRIPTION
**Pull request by idwilliams**
﻿

### Changes proposed in this pull request

- Adding the module mapping for Microsoft 365 Lighthouse (tenantRelationships/managedTenants)

### CODE REVIEW

**Review by timayabi2020 — APPROVED** [review]
Thanks for your contribution :)

File: config/ModulesMapping.jsonc [hunk 1]
```diff
@@ -23,6 +23,7 @@
   "Identity.Governance": "^accessReviews\\.|^businessFlowTemplates\\.|^programs\\.|^programControls\\.|^programControlTypes\\.|^privilegedRoles\\.|^privilegedRoleAssignments\\.|^privilegedRoleAssignmentRequests\\.|^privilegedApproval\\.|^privilegedOperationEvents\\.|^privilegedAccess\\.|^agreements\\.|^users.agreementAcceptance$|^identityGovernance\\.",
   "Identity.SignIns": "^organization.certificateBasedAuthConfiguration$|^invitations\\.|^identityProviders\\.|^oauth2PermissionGrants\\.|^identityProtection\\.|^dataPolicyOperations\\.|^identity\\.|^trustFramework\\.|^informationProtection\\.|^policies\\.|^users.authentication$|^users.informationProtection$",
   "Mail": "^users.inferenceClassification$|^users.mailFolder$|^users.message$",
+  "ManagedTenants": "^tenantRelationships/managedTenants$",
```
Comment (peombwa) [inline config/ModulesMapping.jsonc:26]: We use OpenAPI tags to slice the API. 
```suggestion
  "ManagedTenants": "^tenantRelationships.managedTenants$",
```

**Comment by peombwa** [comment]
Thanks for the contribution!

@timayabi2020, see https://docs.microsoft.com/en-us/graph/api/resources/managedtenants-managedtenant?view=graph-rest-beta.

**Comment by idwilliams** [comment]
@peombwa and @timayabi2020 thank you for taking the time to review this request. I have updated the request to use OpenAPI tags as requested.

**Review by peombwa — APPROVED** [review]

**Pull request merged**